packages/progress-bar/README.md

```diff
@@ -1,33 +1,44 @@
-# @acpaas-ui/progress-bar
+# @acpaas-ui/ngx-components/progress-bar
+
 This module provides a progress bar with a max value.
 
-## Installation
-```
-npm install @acpaas-ui/progress-bar
-```
+## Usage
 
-Import component in **app.module.ts**
+```typescript
+import { ProgressBarModule } from '@acpaas-ui/ngx-components/progress-bar'`;
 ```
-import { ProgressBarModule } from '@acpaas-ui/progress-bar';
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### Progress bar module
+
+#### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() value: number;` | `0` | The current value of the progress. |
+| `@Input() max: number;` | `0` | The value when the progress is completed. |
+
+#### Example
+
+```typescript
+import { ProgressBarModule } from '@acpaas-ui/ngx-components/progress-bar';
 
 @NgModule({
-    imports: [
-        ProgressBarModule
-    ]
-})
+	imports: [
+		ProgressBarModule
+	]
+});
 
-export class AppModule {}
+export class AppModule {};
 ```
 
-## Usage
-```
+```html
 <aui-progress-bar [value]="uploadProgress" max="100"></aui-progress-bar>
```

Comment:
This example is not quite complete. `uploadProgress` is nowhere to be found. To give the reader a nice working example, I would include both `uploadProgress` and `max` in a separate piece of controller code. Also, don't forget to adjust this example in the demo too.